docs/index.md

```diff
@@ -0,0 +1,18 @@
+# Unlock Documentation
+
+[Unlock](https://unlock-protocol.com) is an **access control protocol** that enables creators 
+to monetize their work directly - without relying on a middleman.
+
+Consumers manage all of their subscriptions in a consistent way, as well as earn discounts when 
+they share the best content and applications they use.
+
+It's all built on a blockchain, so it's decentralized and free to use forever.
+
+## Using Unlock
+
+* [Installing Unlock](installation/index.md)
```

Comment:
Hey! I cannot see this file? Let's not link to it until it's there (and put a note...)